e2e_tests/test_browse.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from playwright.sync_api import Page
+
+username = os.getenv("AYR_TEST_USERNAME")
+password = os.getenv("AYR_TEST_PASSWORD")
+
+
+def test_has_title(page: Page):
+    page.goto("/browse")
+    page.get_by_label("Email address").click()
+    page.get_by_label("Email address").fill(username)
+    page.get_by_label("Password").click()
+    page.get_by_label("Password").fill(password)
+    page.get_by_role("button", name="Sign in").click()
+    assert page.title() == "Browse – AYR - Access Your Records – GOV.UK"
+
+
+def test_sort_dropdown(page: Page):
+    page.goto("/browse")
+    page.get_by_label("Email address").click()
+    page.get_by_label("Email address").fill(username)
+    page.get_by_label("Password").click()
+    page.get_by_label("Password").fill(password)
+    page.get_by_role("button", name="Sign in").click()
+    page.get_by_label("", exact=True).click()
+    page.get_by_role("button", name="Search").click()
+    page.select_option("#sort", "body-a")
+    page.select_option("#sort", "body-b")
+    page.select_option("#sort", "series-a")
+    page.select_option("#sort", "series-b")
+    page.select_option("#sort", "date-first")
+    page.select_option("#sort", "date-last")
+
+
+def test_page_loading(page: Page):
+    page.goto("/browse")
+    page.get_by_label("Email address").click()
+    page.get_by_label("Email address").fill(username)
+    page.get_by_label("Password").click()
+    page.get_by_label("Password").fill(password)
+    page.get_by_role("button", name="Sign in").click()
+    page.wait_for_selector(".govuk-heading-l")
```

Comment:
what are we testing here?

Reply:
This was to test that the page loads, it is a bit of a redundant test as it should be covered by other tests in the suite.

Reply:
I've removed this test for now